src/components/Articles/Article.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react'
+import {getArticle} from "../../redux/actioncreators/getArticleActions"
+import {connect} from 'react-redux'
+import {Segment, Loader, Container} from 'semantic-ui-react';
+
+export class Article extends Component {
+    componentDidMount() {
+        this
+            .props
+            .getArticle()
```

Comment:
This is not good practice. 
It should be `this.props.getArticle()` in one line, for good readability.

Reply:
Okay, let me correct that. The beautify plugin changed it to that on save